ReachabilityUI/ReachabilityUI/Reachability/ReachabilityModels.swift

```diff
@@ -23,51 +23,138 @@ extension Reachability.ReachabilityListener {
+                finalOptionsDict[.noConnectionTitleColor] = noConnectionTitleColor
+            } else {
+                finalOptionsDict[.noConnectionTitleColor] = defaultOptionsDict[.noConnectionTitleColor]
+            }
+            
+            //noConnectionBackgroundColor
+            if let noConnectionBackgroundColor = options[.noConnectionBackgroundColor] as? UIColor {
+                finalOptionsDict[.noConnectionBackgroundColor] = noConnectionBackgroundColor
+            } else {
+                finalOptionsDict[.noConnectionBackgroundColor] = defaultOptionsDict[.noConnectionBackgroundColor]
+            }
+            
+            //backgroundColor
+            if let backgroundColor = options[.backgroundColor] as? UIColor {
+                finalOptionsDict[.backgroundColor] = backgroundColor
+            } else {
+                finalOptionsDict[.backgroundColor] = defaultOptionsDict[.backgroundColor]
+            }
+            
+            //height
+            if let height = options[.height] as? CGFloat {
+                finalOptionsDict[.height] = height
+            } else {
+                finalOptionsDict[.height] = defaultOptionsDict[.height]
+            }
+            
+            //font
+            if let font = options[.font] as? UIFont {
+                finalOptionsDict[.font] = font
+            } else {
+                finalOptionsDict[.font] = defaultOptionsDict[.font]
+            }
+            
+            //textAlignment
+            if let textAlignment = options[.textAlignment] as? NSTextAlignment {
+                finalOptionsDict[.textAlignment] = textAlignment
+            } else {
+                finalOptionsDict[.textAlignment] = defaultOptionsDict[.textAlignment]
+            }
+            
+            //animation
+            if let animation = options[.animation] as? Animation {
+                finalOptionsDict[.animation] = animation
+            } else {
+                finalOptionsDict[.animation] = defaultOptionsDict[.animation]
+            }
+            
+            //appearance
+            if let appearance = options[.appearance] as? Appearance {
+                finalOptionsDict[.appearance] = appearance
+            } else {
+                finalOptionsDict[.appearance] = defaultOptionsDict[.appearance]
+            }
+            
+            //appearanceAdjustment
+            if let appearanceAdjustment = options[.appearanceAdjustment] as? CGFloat {
+                finalOptionsDict[.appearanceAdjustment] = appearanceAdjustment
+            } else {
+                finalOptionsDict[.appearanceAdjustment] = defaultOptionsDict[.appearanceAdjustment]
+            }
+            
+        } else {
+            finalOptionsDict = defaultOptionsDict
+        }
+        
+        self.options = finalOptionsDict
     }
 }
 
 extension ReachabilityConfiguration {
     
+    public enum Key: String {
+        case titleColor //Has to be of type UIColor
```

Comment:
Change from `//` to `///` on these comments. That way the comment will be part of the generated documentation